tools/introspection/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ set(${PROJECT_NAME}_DIR ${CMAKE_CURRENT_LIST_DIR}/cmake
     FORCE
 )
 
+# This is required for mac build (set to c++17 globally)
+add_compile_options(-std=c++11)
```

Comment:
@elfenpiff didn't we find another solution for this? e.g. like let the linked library pass this?

Reply:
We have to set the C++ standard via the property CXX_STANDARD for every target like this here: https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD.html